woocommerce/action-scheduler
# change all _ in FQ file names to -

Closes #614 

The file name changes in this PR are to address [this vulnerability](https://stackoverflow.com/a/30913177).

### Testing Instructions

1- Go to Tools -> Scheduled Actions -> Pending
2- Add this test snippet
```
add_action( 'init', function() {
    for ( $i = 1; $i < 200; $i++ )
        as_enqueue_async_action( 'as-async-filename' . $i );
} );
```
3- Refresh the screen
  - You should have 200 pending actions

4- Remove the snippet
5- Wait up to a minute to allow the async runner to run
6- Check that the 200 actions created by the snippet were processed
7- Run `wp help action-scheduler` to verify WP CLI code is loading correctly
